angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -47,8 +47,9 @@
 			      <span class="fa fa-plus"></span> Add to home
 			    </button>
 
-		    <!-- Launch button with access -->
-		    <a class="btn btn-success" href="{{::portlet.maxUrl}}" target="{{::portlet.target}}" ng-if="portlet.canAdd"><span class="fa fa-arrow-circle-o-right"></span> Launch</a>
+		    <sub class="sr-only" id="goTo-portlet.title-{{::portlet.fname}}">go to this app</sub>
```

Comment:
sounds great. Will do it.

On Mon, Aug 10, 2015 at 5:43 PM, Andrew Petro <notifications@github.com>
wrote:

> In
> angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/297#discussion_r36693720>
> :
>
> > @@ -47,8 +47,9 @@
> >  			      <span class="fa fa-plus"></span> Add to home
> >  			    </button>
> >
> > -		    <!-- Launch button with access -->
> > -		    <a class="btn btn-success" href="{{::portlet.maxUrl}}" target="{{::portlet.target}}" ng-if="portlet.canAdd"><span class="fa fa-arrow-circle-o-right"></span> Launch</a>
> > +		    <sub class="sr-only" id="goTo-portlet.title-{{::portlet.fname}}">go to this app</sub>
>
> How about go to {{::portlet.title}} rather than go to this app? Seems
> like a worrying use of the term "app". [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/297/files#r36693720>
> .
>